.travis.yml

```diff
@@ -12,8 +13,10 @@ rvm:
   - 2.6.5
 bundler_args: --without production
 addons:
-    postgresql: "9.6"
+    postgresql: "13"
```

Comment:
I'd like to keep this compatible with pg down to 9.6 for the moment. The issue, I imagine, is that since the dump is in SQL (rails doesn't support pgsql enums properly), the dump itself is incompatible with 9.6. But I imagine with the changes I added to the dump, it should work with 9.6 too :)